src/components/custom/other-platforms.js

```diff
@@ -430,8 +430,6 @@ export const NavResources = ({ onClick }) => (
             onClick={onClick}
             to={community_url}
             target="_blank"
-            external="true"
```

Comment:
this is not the cause of the problem, the issue is happening on loalized-link.js, afaik, in the previous release we have include the fix for this, please check again in production if this issue is still happening 🙏